bundles/org.openhab.binding.mynice/README.md

```diff
@@ -0,0 +1,114 @@
+# MyNice Binding
+
+This binding implements the support of the IT4Wifi module through the NHK protocol and enables management of Nice gates automatisms.
```

Comment:
Are you sure you mean "automatism", i.e. "an involuntary piece of behaviour over which an individual has no control."? 😄